docs/device-agent/install.md

```diff
@@ -125,6 +125,12 @@ For backwards compatibility we will continue to publish to both the old and
 new locations for a period of time, but we strongly encourage users to update to the
 new package to ensure you continue to receive the latest updates.
 
+### Upgrading to Device Agent 3.x
+
+Version 3.x of the Device Agent requires Node.js 18 as a minimum; older versions of Node.js are no longer supported.
+
+The `latest` tagged Docker image is now based on Node.js 18.
+
```

Comment:
No comment about installing the older version? Or just trying to keep it simple?

Reply:
I bounced around on it... will have a second look.

Reply:
@hardillb added a comment